src/main/resources/data-mapping.json

```diff
@@ -716,12 +762,95 @@
           "dataSource": {
             "default": "100"
           }
-        }
+        },
+        {
+          "field": "USER_LIMIT",
+          "dataSource": {
+            "from": "//PurchaseOption/Code",
+            "translation": "getPurchaseOptionCode"
+          }
+        }          
       ]
     },
    {
       "orderType": "ListedPrintSerial",
       "mappings": [
+        {
+          "field": "MATERIAL_TYPE",
```

Comment:
The local data fields will not be present in the listed print serial GOBI order xml. Same with all the below fields that are added additionally